src/msw/window.cpp

```diff
@@ -1996,6 +2001,21 @@ wxWindowMSW::DoMoveSibling(WXHWND hwnd, int x, int y, int width, int height)
     return false;
 }
 
+void wxWindowMSW::MoveOutsideShortInterval(WXHWND hwnd, int x, int y, int width, int height)
+{
+    // scroll to the actual position (looks like there is no need to Freeze() the parent)
+    ::ScrollWindow(GetParent()->GetHWND(), -x, -y, NULL, NULL);
```

Comment:
In principle, we're supposed to pass `GetHwndOf(GetParent())` to the native functions as this macro returns `HWND` and not `WXHWND`.